tests/test_accuracy.py

```diff
@@ -53,10 +54,10 @@ def heat(n, deg, time_stages):
     return errornorm(uexact, u) / norm(uexact)
 
 
-@pytest.mark.parametrize(('deg', 'convrate', 'time_stages'),
-                         [(1, 1.78, i) for i in (1, 2)]
-                         + [(2, 2.8, i) for i in (2, 3)])
-def test_heat_eq(deg, convrate, time_stages):
+@pytest.mark.parametrize(('deg', 'convrate', 'time_stages', 'splitting'),
+                         [(1, 1.78, i, splt) for i in (1, 2) for splt in (AI, IA)]
+                         + [(2, 2.8, i, splt) for i in (2, 3) for splt in (AI, IA)])
+def test_heat_eq(deg, convrate, time_stages, splitting):
```

Comment:
FWIW, a much nicer way of doing this is to just use multiple `parameterize` markers:
```
@pytest.mark.parametrize("splitting", (AI, IA))
@pytest.mark.parametrize("deg", "convrate", "time_stages", ...)
def test_heat_eq(deg, convrate, time_stages, splitting):
   ...
```

Reply:
Yes, thanks, fixed.